### DISCUSSION TITLE
Installed Visual Studio Build Tools but cannot find cl.exe

### DISCUSSION THREAD

**Discussion by kevin-guimard-ext** [microsoft/vscode-cpptools#11553] [Q&A]
Hello,

I installed MSVC v142 - VS 2019 and the latest SDK from the Visual Studio Installer. First, for some reason, the installation is named "VS 2022". Anyway, I ran the Developer Command Prompt but cannot find cl.exe. Why?

Thanks,
Kevin

**Comment by adaml-at-eargo** [comment]
Do you run VS Code from the Developer Command Prompt? What indicates that `cl.exe` can't be found? For example, does `where cl.exe` return a sensible result?

**Comment by kevin-guimard-ext** [comment]
The problem has been solved following a question in another forum. I had to explicitly install VS 2019 from an old installer.
